spec/message_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'message'
+
+describe Message do
+  describe '.all' do
```

Comment:
Syntax for unit tests - do you know when we should write .all and when we should write #all ? 

I feel like they are the same but don't know what the difference is!